js-weather/weather.js

```diff
@@ -0,0 +1,48 @@
+//関数リテラルは呼び出し元のコードよりも先に記述する
+const getWeather = (data) => {
+    if (data === null) {
+        //<div id = "result">HTMLElementオブジェクトを生成
+        const result = document.getElementById('result');
+        result.textContent = '天気を取得できませんでした';
+    } else {
+        // 天気と気温を取得し、配列にする
+        const weatherConditions = data.weather[0].description;
+        const temp = data.main.temp;
+        const MaxTemp = data.main.temp_max;
+        const MinTemp = data.main.temp_min;
+        const array = [weatherConditions, temp, MaxTemp, MinTemp];
+        const weatherInfo = document.getElementsByClassName('weatherInfo');
+        for (let i = 0, len = weatherInfo.length; i < len; i++) {
+            const value = document.createTextNode(array[i]);
+            weatherInfo[i].appendChild(value);
+        }
+    }
+}
+
+// htmlファイルロードとき
+document.addEventListener('DOMContentLoaded', () => {
+    // <script>タグを生成
+    const scr = document.createElement('script');
```

Comment:
（参考）
あまり略称は使わないほうが良いです。
コードが長くなったりすると、可読性が重要になるので、意識できると良いと思います。

あと仮に`script`という変数名にしてしまうと、=でつないだ左辺と右辺が透過関係でなくなります。
例えば、こんな感じでしょうか。（ちょっとくどいけど）

```suggestion
    const scriptElement = document.createElement('script');
    とか
    const scriptDom = document.createElement('script');
```